src/blog/high-performance-cpp-gc.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 'High-performance garbage collection for C++'
+author: 'Anton Bikineev, Omer Katz ([@omerktz](https://twitter.com/omerktz)), and Michael Lippautz ([@mlippautz](https://twitter.com/mlippautz)), C++ memory whisperers'
+avatars:
+  - 'anton-bikineev'
+  - 'omer-katz'
+  - 'michael-lippautz'
+date: 2020-05-15
+tags:
+  - memory
+  - internals
+  - cppgc
+description: 'This post describes the Oilpan C++ garbage collector, its usage in Blink, and how it optimizes sweeping, i.e., reclamation of unreachable memory.'
+---
+
+In the past we have [already](https://v8.dev/blog/trash-talk) [been](https://v8.dev/blog/concurrent-marking) [writing](https://v8.dev/blog/tracing-js-dom) about garbage collection for JavaScript, the document object model (DOM), and how all of this is implemented and optimized in V8.
+Not everything in Chromium is JavaScript though, as most of the browser and its Blink rendering engine where V8 is embedded are written in C++.
```

Comment:
Note that this just inserts newlines, not new paragraphs - for them you need an empty line between. I recommend editing markdown with `npm run serve` so that you could see all the changes live the way they will be rendered.

Also, a paragraph per each sentence is probably an overkill. I'd leave those together where it makes sense semantically.

Reply:
Thanks for explaining offline the newline handling on the blog.

- Removed the newlines again
- Added one more actual paragraph in the intro.